airflow/providers/google/marketing_platform/operators/display_video.py

```diff
@@ -336,3 +336,86 @@ def execute(self, context: Dict):
             self.params,
         )
         hook.run_query(query_id=self.report_id, params=self.params)
+
+
+class GoogleDisplayVideo360SDFtoGCSOperator(BaseOperator):
+    """
+    Download data from SDF and upload it to the Big Query.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GoogleDisplayVideo360SDFtoGCSOperator`
+
+    .. seealso::
+        Check also the official API docs:
+        `https://developers.google.com/display-video/api/reference/rest`
+
+    :param version: The SDF version of the downloaded file..
+    :type version: str
+    :param partner_id: The ID of the partner to download SDF for.
+    :type partner_id: str
+    :param advertiser_id: The ID of the advertiser to download SDF for.
+    :type advertiser_id: str
+    :param parent_entity_filter: Filters on selected file types.
+    :type parent_entity_filter: Dict[str, Any]
+    :param id_filter: Filters on entities by their entity IDs.
+    :type id_filter: Dict[str, Any]
+    :param inventory_source_filter: Filters on Inventory Sources by their IDs.
+    :type inventory_source_filter: Dict[str, Any]
+    :param gcp_conn_id: The connection ID to use when fetching connection info.
+    :type gcp_conn_id: str
+    :param delegate_to: The account to impersonate, if any. For this to work, the service account making the
+        request must have  domain-wide delegation enabled.
+    :type delegate_to: str
+    """
+
+    template_fields = ("bucket_name", "object_name")
+
+    @apply_defaults
+    def __init__(
+        self,
+        body_request: Dict[str, Any],
+        bucket_name: str,
+        object_name: str,
+        gzip: bool = False,
+        api_version: str = "v1",
+        gcp_conn_id: str = "google_cloud_default",
+        delegate_to: Optional[str] = None,
+        *args,
+        **kwargs,
+    ) -> None:
+        super().__init__(*args, **kwargs)
+        self.body_request = body_request
+        self.bucket_name = bucket_name
+        self.object_name = object_name
+        self.gzip = gzip
+        self.api_version = api_version
+        self.gcp_conn_id = gcp_conn_id
+        self.delegate_to = delegate_to
+
+    def execute(self, context: Dict):
+        hook = GoogleDisplayVideo360Hook(
+            gcp_conn_id=self.gcp_conn_id,
+            delegate_to=self.delegate_to,
+            api_version=self.api_version,
+        )
+        gcs_hook = GCSHook(
+            google_cloud_storage_conn_id=self.gcp_conn_id, delegate_to=self.delegate_to
+        )
+
+        self.log.info("Creating operation for SDF download task...")
+        operation_name = hook.create_sdf_download_operation(body_request=self.body_request)
+        operation = hook.get_sdf_download_operation(name=operation_name)
```

Comment:
It's ok, however I would expect `hook.create_sdf_download_operation` to return an operation object. But as I said, I am ok with this too

Reply:
Ok.